mnefun/_mnefun.py

```diff
@@ -1246,6 +1254,13 @@ def run_sss_locally(p, subjects, run_indices):
             raw = read_raw_fif(r, preload=True, allow_maxshield='yes')
             raw.fix_mag_coil_types()
             _load_meg_bads(raw, prebad_file, disp=ii == 0, prefix=' ' * 6)
+            # If so run Maxfilter for automatic bad channel selection
+            if p.mf_autobad:
+                print('     Maxfilter auto bad channel selection requested. '
+                      'Running on kasga...', end='')
+                add_bads = do_mf_autobad(p, r, prebad_file)
```

Comment:
I think this should write to a file, and only run `maxfilter` if the file does not exist. It makes it so that you only ever need to run it once, even if you run `sss` multiple times.

Reply:
Okay